packages/devextreme/js/__internal/scheduler/appointment_popup/m_popup.ts

```diff
@@ -115,6 +118,11 @@ export class AppointmentPopup {
       onShowing: (e): void => this._onShowing(e),
       wrapperAttr: { class: APPOINTMENT_POPUP_CLASS },
     };
+
+    return {
+      ...defaultPopupConfig,
+      ...userPopupOptions,
+    };
```

Comment:
We need a deep extend of options here:
```js
extend(true, defaultPopupConfig, userPopupOptions);
```

Reasons are the same as [here](https://github.com/DevExpress/DevExtreme/pull/31555#discussion_r2486312229), but in this case because of [wrapperAttr](https://github.com/DevExpress/DevExtreme/pull/31530/files/6cfc79f52c07f637a2ad82cf753b1965ee7fb1f4#diff-1b3c3b914f817852cdf156470fdf611795b3a65fc4411fa7a8fba800603a937dR119)